### ISSUE TITLE
PPC Trigger: Blank Story

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#7138]
## Description

When a user creates a new story it is blank and we should not display any messages or warnings on the pre publish checklist. 

## Implementation Suggestions 
- We should create an `isEmpty` property on story data before tackling these triggers (#7145). This will allow us to trigger the PPC when we need it while keeping the logic for what an "empty" story is out of the checklist and in the story itself. An empty story is any story with just a background color set. 

## Acceptance Criteria 
- When a user creates a new story don't trigger any PPC suggestions. No more icon next to checklist title in panel tab, no icon on publish button. Both high priority and recommended tabs of ppc are disabled. Accordion arrows are gone.
- Should also include updated unit tests 

https://www.figma.com/file/1YM3jA9h4QGc4xLdyaXIJS/Stories-Sprint-1.7?node-id=7045%3A7520
![Screen Shot 2021-04-07 at 3 18 56 PM](https://user-images.githubusercontent.com/10720454/113941774-98e53e00-97b4-11eb-91e3-fbbe76796f6a.png)

**Comment by maxyinger** [comment]
I feel like this would mostly get taken care of in this PR with the proof of concept being firing an event `onDirty` for the story here:
https://github.com/google/web-stories-wp/pull/7259

Then the rest by tying in Brittany's state machine for PPC PR here:
https://github.com/google/web-stories-wp/pull/7241#discussion_r616026968

**Comment by BrittanyIRL** [comment]
@littlemilkstudio  - agree, this ticket feels like the natural test case for the implementation.

**Comment by BrittanyIRL** [comment]
@csossi - next time you do QA for pea pod please do this one first, we need it to unblock 2 other tickets. (this pr: https://github.com/google/web-stories-wp/pull/7297)

**Comment by csossi** [comment]
Added an element and now seeing the "Recommended" messages

Deleted the element to return to a story with no content. Messages remained - shouldn't they be removed at this point?

**Comment by BrittanyIRL** [comment]
@csossi - the enabling will not revert. Once a section is enabled it should stay enabled.

**Comment by csossi** [comment]
Verified in QA
